.github/workflows/update-browserlist.yml

```
@@ -9,7 +9,7 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v4
-      - run: npx browserslist@latest --update-db
+      - run: npm install pnpm -g && npx update-browserslist-db@latest
```

```suggestion
      - run: npx update-browserslist-db@latest
```

What error is happening with this?

---

Package sees pnpm-lock file and tries to use pnpm, which is not installed in the environment.